src/UI/Localization.cpp

```
@@ -0,0 +1,146 @@
+			return bytes;
+		}
+
+		bool has(const std::vector<std::string>& v, const std::string& s)
+		{
+			return std::find(v.begin(), v.end(), s) != v.end();
+		}
+
+		// Choose the best available locale file for a wanted code: an exact match ("fr_fr"), else the base
+		// language of a regional code ("fr_fr" -> "fr"), else any regional variant of a base code
+		// ("fr" -> "fr_ca"). Empty string means "no file, use English".
+		std::string bestMatch(const std::string& want, const std::vector<std::string>& avail)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)